src/component/Header/Header.module.css

```diff
@@ -0,0 +1,20 @@
+.pomo_logo {
```

Comment:
BEM style dictates that we use `-` instead of `_` for separators. Change all your class names to use `-` instead of `_`.